source/regression2.md

```diff
@@ -305,13 +305,13 @@ small_sacramento_pred = small_sacramento_pred[["sqft", "price", "predicted"]].me
     id_vars=["sqft"]
 )
 
-error_plot = small_plot.copy()
-
+v_lines = []
 for i in range(len(small_sacramento)):
     sqft_val = small_sacramento.iloc[i]["sqft"]
     line_df = small_sacramento_pred.query("sqft == @sqft_val")
-    error_plot += alt.Chart(line_df).mark_line(color="red").encode(x="sqft", y="value")
+    v_lines.append(alt.Chart(line_df).mark_line(color="red").encode(x="sqft", y="value"))
 
+error_plot = alt.layer(*v_lines, small_plot).configure_circle(opacity=1)
```

Comment:
All the changes in the regression plots is to put the dots on top of the lines which I think looks more appealing

Current:
![image](https://github.com/UBC-DSCI/introduction-to-datascience-python/assets/4560057/5c0c007e-9f3f-455e-9381-250ca36012ff)

Suggested:
![image](https://github.com/UBC-DSCI/introduction-to-datascience-python/assets/4560057/dbfc97c3-dc3f-4d50-9852-4e781121b69e)

Reply:
agreed! nice change